examples/http/tracer.js

```diff
@@ -25,7 +25,7 @@ module.exports = (serviceName) => {
   provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 
   // Initialize the OpenTelemetry APIs to use the BasicTracerProvider bindings
-  opentelemetry.trace.setGlobalTracerProvider(provider);
+  provider.register();
```

Comment:
Should we add it as well to the example in the README ?

Reply:
https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-plugin-http/README.md#usage

Reply:
+1, also getting-started guide?